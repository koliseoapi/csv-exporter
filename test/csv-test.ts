import createCsvExporter from "../src/csv-exporter";
import { config, data } from "./mock-data";

describe("CSV Exporter", () => {
  it("renders CSV", () => {
    const csv = createCsvExporter(config);
    const output = csv.export(data);
    //    console.log(output);
    expect(output).toMatchSnapshot();
  });
});
