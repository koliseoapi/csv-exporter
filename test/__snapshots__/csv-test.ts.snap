// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSV Exporter renders CSV 1`] = `
"name,\\"Years \\"\\"old\\"\\"\\",created,description
\\"Bond, James\\",30,\\"1953-04-13T00:00:00.000Z\\",
Sandokan,-37,\\"1895-01-01T00:00:00.000Z\\",\\"Emilio Salgari wrote several novels chronicling the adventures of Sandokan and Yanez, two of his most legendary creations.
 The pirates are introduced in \\"\\"The Tigers of Mompracem\\"\\", which portrays their relentless struggle against the Dutch and British powers that seek to wipe them out. \\"
\\"Mortadelo y Filem√≥n\\",50.56667,,
,,,
,,,
"
`;
